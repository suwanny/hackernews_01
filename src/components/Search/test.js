import React from 'react';
import ReactDOM from 'react-dom';
import Search from '.';
import renderer from 'react-test-renderer';

describe('Search', () => {
  it('renders', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Search>Search</Search>, div);
  });

  test('snapshots', () => {
    const component = renderer.create(
      <Search>Search</Search>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
