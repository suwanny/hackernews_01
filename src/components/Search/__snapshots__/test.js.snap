// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshots 1`] = `
<form
  onSubmit={undefined}
>
  <input
    onChange={undefined}
    type="text"
    value={undefined}
  />
  <button
    type="submit"
  >
    Search
  </button>
</form>
`;
